c2_components/src/mfx_c2_decoder_component.cpp

```diff
@@ -40,6 +40,10 @@ using namespace android;
 #define MFX_DEBUG_MODULE_NAME "mfx_c2_decoder_component"
 
 const c2_nsecs_t TIMEOUT_NS = MFX_SECOND_NS;
+const uint64_t kMinInputBufferSize = 2 * WIDTH_2K * HEIGHT_2K;
```

Comment:
Is the bitstream size, how the function of 2*w*h come from?

Reply:
It is referred from software codec2.0 https://cactus-source.iind.intel.com/source/xref/CELADON-MASTER-R2/frameworks/av/media/codec2/components/avc/C2SoftAvcDec.cpp#33.
As our hardware codec2.0 support 4k/8k, refined it here.

Reply:
For decoder this parameter is indiate the input bitstream size right? but I think 2x2Kx2K may too big for it. 　according the framework, if we just ignore it or set it to 0, framework can also calculate it, any side effect if just leave it as blank?
1245        if (maxInputSize.value == 0) {
1246            if (config->mDomain & Config::IS_AUDIO) {
1247                maxInputSize.value = encoder ? 16384 : 4096;
1248            } else if (!encoder) {
1249                maxInputSize.value = 1048576u;
1250            }
1251        }
to line 1285

Reply:
Yes, it is needed and cannot be set as blank. Otherwise, we will face input buffer capacity issue when seeking very frequently and video cannot be played.